modules/installation-custom-aws-vpc.adoc

```diff
@@ -14,12 +15,20 @@ endif::[]
 ifeval::["{context}" == "installing-aws-secret-region"]
 :aws-secret:
 endif::[]
+ifeval::["{context}" == "installing-aws-outposts-remote-workers"]
+:aws-outposts:
+endif::[]
 
 :_content-type: CONCEPT
 [id="installation-custom-aws-vpc_{context}"]
 = About using a custom VPC
 
+ifndef::aws-outposts[]
 In {product-title} {product-version}, you can deploy a cluster into existing subnets in an existing Amazon Virtual Private Cloud (VPC) in Amazon Web Services (AWS). By deploying {product-title} into an existing AWS VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. If you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself, use this installation option.
+endif::aws-outposts[]
+ifdef::aws-outposts[]
+{product-title} {product-version} installer cannot automatically deploy AWS Subnets on AWS Outposts, so you will need to manually configure the VPC. Therefore you have to deploy the cluster into existing subnets in an existing Amazon Virtual Private Cloud (VPC) in Amazon Web Services (AWS). In addition, by deploying {product-title} into an existing AWS VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company’s guidelines set.
+endif::aws-outposts[]
 
 Because the installation program cannot know what other components are also in your existing subnets, it cannot choose subnet CIDRs and so forth on your behalf. You must configure networking for the subnets that you install your cluster to yourself.
```

Comment:
"install your cluster to" - do you mean "install your cluster on"?

Reply:
Please notice - this part wasn't added for AWS Outposts, reusing existing text